systems/citybuilding.go

```diff
@@ -5,39 +5,46 @@ import (
 
 	"engo.io/ecs"
 	"engo.io/engo"
+	"engo.io/engo/common"
 )
 
-type MouseTracker struct {
+// ---------------------------------
+// Vars
+
+// CityAssets are the assets for the system
+var CityAssets = []string{"textures/city.png"}
```

Comment:
I don't think this is needed. It's just one item in an array that's not changing.